src/create-http-client.js

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs';
+import https from 'https';
+import path from 'path';
+import axios from 'axios';
```

Comment:
Would it be possible to use the `unfech` module for fetching? It's only 518B, while Axios is 4.16k.